avocado/plugins/human.py

```diff
@@ -99,3 +100,22 @@ def post_tests(self, job):
                       job.result.warned, job.result.interrupted,
                       job.result.cancelled)
         self.log.info("TESTS TIME : %.2f s", job.result.tests_total_time)
+
+
+class HumanJob(JobPre, JobPost):
+
+    """
+    Human result UI
+    """
+
+    name = 'human'
+    description = "Human Interface UI"
+
+    def pre(self, job):
+        pass
+
+    def post(self, job):
+        if job.time_elapsed != -1:
+            if not getattr(job.args, 'stdout_claimed_by', None):
+                log = logging.getLogger("avocado.app")
+                log.info("JOB TIME   : %.2f s", job.time_elapsed)
```

Comment:
I'd suggest removing the `TESTS TIME` as it's not really needed and only makes the output unnecessary longer.